src/index.js

```diff
@@ -1,50 +1,27 @@
-// import { score , storeScore } from './modules/storage';
+import { getScores, addScores } from './modules/game-api.js';
```

Comment:
When starting the app initially, the scores should already be populated on the UI. The refresh button should only be used after posting a score to re-fetch the data and populate the UI. However, in your case, when starting the app, there is no score on the UI. Instead refresh button needs to be clicked to get the score. Kindly fix this issue.